mmv1/templates/terraform/state_migrations/cloud_build_trigger.go.erb

```diff
@@ -373,8 +373,7 @@ The elements are of the form "KEY=VALUE" for the environment variable "KEY" bein
 									"machine_type": {
 										Type:         schema.TypeString,
 										Optional:     true,
-										ValidateFunc: verify.ValidateEnum([]string{"UNSPECIFIED", "N1_HIGHCPU_8", "N1_HIGHCPU_32", "E2_HIGHCPU_8", "E2_HIGHCPU_32", ""}),
```

Comment:
I don't think this file is needed to change, as it is the old schema of the previous schema version.

Reply:
Reverted the chance.